src/lib/Quiz.jsx

```diff
@@ -108,6 +110,16 @@ function Quiz({
       return false;
     }
 
+    if (enableProgressBar && typeof enableProgressBar !== 'boolean') {
+      console.error('enableProgressBar must be a Boolean');
+      return false;
+    }
+
+    if (progressBarColor && typeof progressBarColor !== 'string') {
```

Comment:
what if i input a not-color code? e.g. `abc`

Reply:
Hi, i added a function that verifies the hex value for the progress bar.